tests/serialise/xml-serialise/run.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+import os.path
+import unittest
+from decimal import Decimal
+
+import six
+from bitarray import bitarray
+from lxml import objectify
+
+from pyangbind.lib.serialise import pybindIETFXMLEncoder
+from pyangbind.lib.xpathhelper import YANGPathHelper
+from tests.base import PyangBindTestCase
+
+
+def xml_tree_equivalence(e1, e2):
```

Comment:
Perhaps we can use http://python-xmlunittest.readthedocs.io/en/latest/xmlunittest.html#xml-documents-comparison-assertion for this test -- it seems easier than maintaining an XML equivalence implementation ourselves here?

Reply:
OK, sure! Let's go with this one then :-)